src/pages/privacy-policy.tsx

```diff
@@ -0,0 +1,22 @@
+import Layout from "@src/components/Layout/index";
+import {
+  Section,
+  SectionContent,
+  SectionWrapper,
+} from "@src/const/styles/pages/index";
+import { Color } from "@src/const/styles/variables";
```

Comment:
Why `"` is used instead of "'"? Can we add a rule to the linter?

Reply:
I want to merge this quickly but will add this in the future